backend/tests/Feature/UserInboxTest.php

```diff
@@ -297,15 +297,20 @@ public function testCreateSaveSendDraft()
 
         //assign user2 to the thread.
         $data['body'] = '<p>newhtml</p>';
-        $data['action'] = 'send';
-        $response = $this->json('PUT', "/drafts/{$data['id']}", $data, ['Authorization' => 'Bearer '.$token]);
+        $response = $this->json('PUT', "/drafts/{$data['id']}/send", $data, ['Authorization' => 'Bearer '.$token]);
         $response
             ->assertStatus(200)
             ->assertJson([
                 'success' => true,
             ]);
         $this->assertDatabaseHas('drafts', ['thread_id' => $thread_id, 'body' => $data['body']]);
         $inbox->delete();
+
```

Comment:
@MaxLinCode this fails because this test function calls the send action endpoint, not the delete action.
You'll need to make a new test function, and call `$this->json('PUT', "/drafts/{$data['id']}/delete"` and then do the `assertDatabaseMissing` as such